spec/rspec/mocks/matchers/receive_spec.rb

```diff
@@ -267,6 +267,14 @@ def receiver.method_missing(*a); end # a poor man's stub...
         it_behaves_like "resets partial mocks cleanly" do
           let(:target) { allow(object) }
         end
+
+        context 'ordered with receive counts' do
+          specify 'is not supported' do
```

Comment:
Why `specify` and not `it`?

Generally, I stick to `it` when the string starts with a verb (as that makes grammatical sense) but use `specify` when my doc string starts with something else (e.g. a noun) as such phrasing makes no sense off of `it`.